### PR TITLE
Improve status action to enable abbreviated responses when only the version is needed

### PR DESCRIPTION
**Pull request by nickwilliams-eventbrite**
Also improves Travis build and Setup files to include proper information, release binary distribution wheels.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
         'For more, see https://github.com/eventbrite/pysoa'
     ),
     url='http://github.com/eventbrite/pysoa',
-    packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
+    packages=list(map(str, find_packages(exclude=['*.tests', '*.tests.*', 'tests.*', 'tests']))),
```
[the remark this answers is not in the document]
Comment (nickwilliams-eventbrite) [reply setup.py]: Learned this the hard way with Invoke Release (haven't figured out why it hasn't bit us in PySOA yet) ... in Python 2, packages must be a list of bytestrings. It barfs if they're unicode strings.

File: pysoa/server/action/status.py [hunk 2]
```diff
@@ -50,6 +50,19 @@ def _version(self):
     def _build(self):
         return None
 
+    request_schema = fields.Dictionary(
+        {
+            'abbreviated_status': fields.Boolean(
```
[the remark this answers is not in the document]
Comment (vartec) [reply pysoa/server/action/status.py]: +1 to changing the name, `abbreviated` isn't immediately obvious w/o reading the description.

File: setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vartec) [reply setup.py]: TIL

File: pysoa/server/action/status.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nickwilliams-eventbrite) [reply pysoa/server/action/status.py]: Done.

**Pull request merged**